options.go

```diff
@@ -27,6 +27,10 @@ type Options struct {
 	// RenameDestination can specify the destination file or dir name if needed to rename.
 	RenameDestination func(src, dest string) (string, error)
 
+	// FileCopyFunc specifies the method by which a regular file is copied.
+	// The default is CopyBytes.
+	FileCopyFunc FileCopyMethod
```

Comment:
What's the purpose of using different names here?
Both the name of prop and the type can be the same. Let me know your thoughts

Reply:
Good point. I'll make them both `FileCopyMethod`